client/signup/steps/jpo-summary/index.jsx

```diff
@@ -0,0 +1,213 @@
+			className="jpo-summary__item-checkmark"
+			size={ 12 }
+			icon="checkmark"
+			/>;
+
+		const {
+			signupProgress: {
+				0: {
+					jpoSiteTitle
+				},
+				1: {
+					jpoSiteType
+				},
+				2: {
+					jpoHomepage
+				},
+				3: {
+					jpoContactForm
+				}
+			}
+		} = this.props;
+
+		const pathPrefix = '/start/jetpack-onboarding/';
+
+		return (
+			<div className="jpo__summary-wrapper">
+				<table>
+					<tbody>
+						<tr>
+							<td>
+								<div className="jpo__summary-col-header jpo-summary__col-header">
+									{
+										translate( "Steps you've completed:" )
+									}
+								</div>
+								<ul className="jpo-summary__completed-onboarding">
+									{
+										jpoSiteTitle
+											? <li>{ checkMark } <span>{ translate( 'Site Title & Description' ) }</span></li>
+											: null
+									}
+									{
+										jpoSiteType
+											? <li>{ checkMark } <span>{ translate( 'Type of Site' ) }</span></li>
+											: null
+									}
+									{
+										jpoHomepage
+											? <li>{ checkMark } <span>{ translate( 'Type of Homepage' ) }</span></li>
+											: null
+									}
+									{
+										jpoContactForm
+											? <li>{ checkMark } <span>{ translate( 'Contact Us Form' ) }</span></li>
+											: null
+									}
+									{
+										connectionToJetpackComplete
+											? <li>{ checkMark } <span>{ translate( 'Jetpack Connection' ) }</span></li>
+											: null
+									}
+								</ul>
+							</td>
+							<td>
+								<div className="jpo__summary-col-header jpo-summary__col-header">
+									{
+										translate( 'Continue your site setup:' )
+									}
+								</div>
+								<ul className="jpo-summary__more-onboarding">
+									{
+										jpoSiteTitle
+											? null
+											: <li><a href={ `${ pathPrefix }jpo-site-title` }>{ translate( 'Site Title & Description' ) }</a></li>
+									}
+									{
+										jpoSiteType
+											? null
+											: <li><a href={ `${ pathPrefix }jpo-site-type` }>{ translate( 'Type of Site' ) }</a></li>
+									}
+									{
+										jpoHomepage
+											? null
+											: <li><a href={ `${ pathPrefix }jpo-homepage` }>{ translate( 'Type of Homepage' ) }</a></li>
+									}
+									{
+										jpoContactForm
+											? null
+											: <li><a href={ `${ pathPrefix }jpo-contact-form` }>{ translate( 'Contact Us Form' ) }</a></li>
+									}
+									{ ! connectionToJetpackComplete ? <li>{ translate( 'Jetpack Connection' ) }</li> : null }
+									<li><a href="#">{ translate( 'Choose a Theme' ) }</a></li>
+									<li><a href="#">{ translate( 'Add a Site Address' ) }</a></li>
+									<li><a href="#">{ translate( 'Add a Store' ) }</a></li>
+									<li><a href="#">{ translate( 'Start a Blog' ) }</a></li>
+								</ul>
+							</td>
+						</tr>
+					</tbody>
+				</table>
+				<div>
+					<Button primary onClick={ this.completeOnboarding }>
+						{
+							translate( 'Visit your site' )
```

Comment:
Hi! I've found a possible matching string that has already been translated **38** times:
`translate( 'Visit Your Site' )` *ES Score: 11*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).